### ISSUE TITLE
CA creation

### ISSUE THREAD

**Issue by r4yfx** [bodsch/docker-icinga2#64]
latest master builds

Error when starting container
```
[2020-01-07 16:27:28] create new CA for 'icinga2'
critical/cli: chown() failed with error code 1, "Operation not permitted"
warning/cli: Cannot set ownership for user 'nagios' group 'nagios' on file '/var/lib/icinga2/certs/'.
critical/SSL: Error while opening private RSA key file '/var/lib/icinga2/certs//6993333bd964.key': 33558541, "error:0200100D:system library:fopen:Permission denied"
critical/cli: Failed to create certificate signing request.
[2020-01-07 16:27:29] ERROR API Setup has failed
[2020-01-07 16:27:29] exit with signal '1'
```

Is this what you were suggesting within your documentation about the SSL certificates?

**Comment by bodsch** [comment]
this issue is related to #63

**Comment by r4yfx** [comment]
thank you for letting me know :)

**Comment by bodsch** [comment]
fixed with fbbca39
